terasoluna-tourreservation-web/src/main/java/org/terasoluna/tourreservation/app/error/ErrorController.java

```diff
@@ -26,7 +28,7 @@
      * Shows the access denied error view.
      * @return access denied error view
      */
-    @RequestMapping("accessDeniedError")
+    @GetMapping("accessDeniedError")
```

Comment:
- [ ] You should consider replacing any RequestMapping attached to the handler method, or not.
I don't recommend implementing it with a partially different policy, just to solve the SonarQube issue.

Reply:
Return to RequestMapping according to the original policy.

d090604be30bf902102e54b48d0e236ca9ac71fd